infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -83,6 +83,14 @@ public void syncEvent(final UInt64 nodeSlot, final UInt64 headSlot, final int nu
     info(syncEventLog, Color.WHITE);
   }
 
+  public void weakSubjectivityFailedEvent(final Bytes32 blockRoot, final UInt64 slot) {
+    final String weakSubjectivityFailedEventLog =
+        String.format(
+            "Sync Event  *** Weak subjectivity check failed block: %s, slot: %s",
+            LogFormatter.formatHashRoot(blockRoot), slot.toString());
+    info(weakSubjectivityFailedEventLog, Color.RED);
```

Comment:
This should probably be at warn level not just info.

Reply:
ok sure - wasn't sure what the events concept was, but i'll bump it.